coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {Card, CardContent, CardActions, Button, Typography} from "@material-ui/core";
+import {humanReadableJoin} from "../format";
+
+function ConnectBackCard(props: { names: string[], tags: string[] }) {
+  return (
+      <Card>
+        <CardContent>
+          <Typography variant="h5" gutterBottom>
+            Connect Back
+          </Typography>
+          <Typography color="textSecondary" gutterBottom>
+            You've recently chatted
+            about {humanReadableJoin(props.tags)} {}
+            with {}
```

Comment:
What do empty {} mean in this syntax?

Reply:
Otherwise there's no space around tags, because JSX transpiler strips spaces

Reply:
But there are no tags on the previous line